lib/src/input/figma/entities/layers/rectangle.dart

```diff
@@ -84,19 +72,27 @@ class FigmaRectangle extends FigmaVector
 
   @override
   Future<PBDLNode> interpretNode() async {
-    var fillsMap =
-        (fillsList == null || fillsList.isEmpty) ? {} : fillsList.first;
-    if (fillsMap != null && fillsMap['type'] == 'IMAGE') {
+    var tempFill;
+
+    try {
```

Comment:
I think this can be simplified to the following:
```
var isTempFill = figmaStyleProperty?.fills?.length == 1 && figmaStyleProperty?.fills?.first?.type == 'IMAGE';
```

If either values end up with null, then `isTempFill` will just end up as false, so no need to check for nullity. However, let me know if I'm missing something.

Reply:
You are correct, thank you!